lib/tasks/publishing_api.rake

```diff
@@ -100,6 +100,13 @@ namespace :publishing_api do
 
       puts "Patching links for #{page.content_id}..."
     end
+
+    root_page = RootBrowsePagePresenter.new('state' => 'published')
+    Services.publishing_api.patch_links(
+      root_page.content_id,
+      root_page.render_links_for_publishing_api
+    )
+    puts "Patching links for root page..."
```

Comment:
The patching has already happened by this point in the code, so maybe put this in the past tense.